webapp/src/dogma/features/auth/authSlice.ts

```diff
@@ -116,28 +133,34 @@ export const authSlice = createSlice({
   reducers: {},
   extraReducers: (builder) => {
     builder
-      .addCase(login.fulfilled, (status, action) => {
-        if (action.payload != null) {
-          status.isAuthenticated = true;
-        }
+      .addCase(login.fulfilled, (state, { payload }) => {
+        state.sessionId = payload.access_token;
+      })
+      .addCase(login.rejected, (state) => {
+        state.sessionId = '';
+      })
+      .addCase(logout.fulfilled, (state) => {
+        state.isInAnonymousMode = true;
+        state.sessionId = '';
+        state.user = null;
+        state.ready = true;
```

Comment:
I'm confused about the meaning of `ready`.
Does it mean the current session is validated?

Reply:
We need to wait until it's ready to check the state. For example, here https://github.com/line/centraldogma/blob/d4c6397baf7367d81677c90cdd5537dbf15ebe62/webapp/src/dogma/features/auth/Authorized.tsx#L41

Reply:
Actually, I was wondering why `ready` is set to `true` for `logout.fulfilled`. From what I understand here, it needs to back to the initial state.

Reply:
For logout, I set it to `true` to bypass the loading state since we don't really need a loading indicator at this point. 
```
 if (!auth.ready) {
    return <p>Loading...</p>;
  }
```
The `ready` state is useful when we have a loading indicator. In that case we may display a progress bar / loading indicator while it's not `ready`.

Reply:
Actually let me remove them for now at it doesn't really matter at this point.